docs/README.md

```diff
@@ -2,27 +2,19 @@
 
 Kubernetes allows for extensions to its architecture in the form of custom
 resources: <https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/>.
-KubeVirt represents virtual machines as custom resources and manages changes
-to libvirt domains based on the state of those resources.
+KubeVirt represents virtual machines as custom resources and manages changes to libvirt domains based on the state of those resources.
```

Comment:
I don't understand why you joined these lines. It makes it quite hard to find which typo you have fixed here.

Reply:
I see, let me revert that

Reply:
The typo was actually regarding the word  `enviornment --> environment`